src/main/webapp/app/entities/submission.model.ts

```diff
@@ -89,6 +91,12 @@ export function getFirstResult(submission: Submission | undefined): Result | und
     }
 }
 
+export function refreshLatestResult(submissions: Submission[]) {
+    submissions.forEach((submission) => {
+        submission.latestResult = getLatestSubmissionResult(submission);
+    });
+}
+
 // todo NR/SE: remove after PR can be merged into develop
```

Comment:
what about this remaining TODO?

Reply:
I missed that one, I removed the funciton and its calls.